### PR TITLE
add kubelet volume size to node definition

### PR DESCRIPTION
**Pull request by teemow**
this makes the size of the kubelet volume that is attached to each node
configurable.

the kubelet uses this volume to store its files and to mount volumes.

disclaimer: the data structure of nodes/workers is messed up already.
there are differences between aws/azure and kvm whether the nodes can be
configured individually or not. this is a mess and this addition will
not aim to fix this. this will be changed with the work on node pools.

built on top of: https://github.com/giantswarm/api-spec/pull/108

### CODE REVIEW

File: definitions.yaml [hunk 1]
```diff
@@ -76,6 +76,23 @@ definitions:
                       default:
                         description: Default size of the docker volume, if not specified
                         type: number
+              kubelet:
+                description: Information about the kubelet volume attached to the worker node. This volume is used to store files for the kubelet (volume mounts etc)
```
Comment (marians) [inline definitions.yaml]: Please append a period to `etc` and also to the end of the sentence.

Comment (marians) [reply definitions.yaml]: How about mentioning that this is also used by `EmptyDir` type volumes?

File: definitions.yaml [hunk 2]
```diff
@@ -241,6 +258,18 @@ definitions:
                 description: |
                   Size of the docker volume attached to the node in GB. Must be
                   the same for all worker nodes of a cluster.
+          kubelet:
+            description: |
+              Volume for the kubelet. Usually mounted at `/var/lib/kubelet`
```
Comment (marians) [inline definitions.yaml]: Please append a period after `/var/lib/kubelet`

File: definitions.yaml (hunk 1 above)
Comment (teemow) [reply definitions.yaml]: added. thanks!

**Pull request closed without merging**